File: server/src/game/tools/conf/output/go/QuotesListConfig.go
package conf

import (
	//"bufio"
	"encoding/json"
	"fmt"
	//"io"
	"io/ioutil"
	"os"
	//"strings"
)

type QuotesListConfigElem struct {
	Id int
	GateType int
	GateId int
	Name string
	DataSeries []int
	Special string
	Textid string
	Des string
	Isbuy string
	
}

type QuotesListConfig0 struct {
	Dic map[int]*QuotesListConfigElem
	Arr []*QuotesListConfigElem
}

var (
	QuotesListConfig QuotesListConfig0
)

func loadQuotesListConfig(path string) {
	file, err := os.Open(path)
	defer func() {
		if file != nil {
			file.Close()
		}
	}()
	if err != nil {
		panic(fmt.Sprint("load config error, path=", path))
	}

	QuotesListConfig.Dic = make(map[int]*QuotesListConfigElem)
	QuotesListConfig.Arr = make([]*QuotesListConfigElem, 0, 32)

	bt,err  :=  ioutil.ReadAll(file)
	json.Unmarshal(bt, &QuotesListConfig.Arr)
	for _,conf := range QuotesListConfig.Arr{
		QuotesListConfig.Dic[conf.Id] = conf
	}
	
}

File: server/src/game/tools/conf/output/ts/QuotesListConfig.ts
class QuotesListConfig {
    private static _dic:Dictionary<QuotesListConfig> = new Dictionary<QuotesListConfig>();
    private static _arr:Array<QuotesListConfig> = new Array<QuotesListConfig>();

	 public Id :number ;
     public GateType :number ;
     public GateId :number ;
    
     public DataSeries : Array<number>  =new Array<number>();
     public Special :string ;
     public Textid :string ;
    
     public Isbuy :string ;
    

    
    public constructor(){

    }


    public static get Dic():Dictionary<QuotesListConfig>{
        return  QuotesListConfig._dic;
    }
    public static get Arr():Array<QuotesListConfig>{
        return QuotesListConfig._arr;
    }
    public static parse(json:any[]){

        for(var i = 0; i < json.length; i++){
            let config:QuotesListConfig = <QuotesListConfig>json[i];
            QuotesListConfig._arr.push(config);
            QuotesListConfig._dic.add(config.Id,config);
        }
    }
}